src/Mod/Draft/DraftGui.py

```diff
@@ -2308,6 +2308,8 @@ def reject(self):
 
 class ShapeStringTaskPanel:
     '''A TaskPanel for ShapeString'''
+    oldValueBuffer = False
```

Comment:
Since this is Python, we should try using PEP8 style, meaning `snake_case` style for variables.

This looks better as `old_value_buffer`.

And no, there is no official coding style, but I and Carlo at least try to be somewhat consistent on this lately.